packages/sit-onyx/src/components/OnyxDialog/OnyxDialog.vue

```diff
@@ -57,18 +56,12 @@ const isExpanded = useVModel({
   key: "open",
   default: false,
 });
-
-const triggerBindings = computed(() => ({
-  onClick: () => (isExpanded.value = !isExpanded.value),
-  "aria-expanded": isExpanded.value,
-  "aria-haspopup": "dialog",
-}));
 </script>
 
 <template>
-  <OnyxBasicPopover :class="['onyx-dialog', densityClass]" v-bind="props" :open="isExpanded">
-    <template #default>
-      <slot name="trigger" :trigger="triggerBindings"> </slot>
+  <OnyxBasicPopover v-bind="props" v-model:open="isExpanded" :class="['onyx-dialog', densityClass]">
+    <template #default="{ trigger }">
+      <slot name="trigger" :trigger="{ ...trigger }"> </slot>
```

Comment:
```suggestion
      <slot name="trigger" :trigger="trigger"> </slot>
```

No need to destructure here :)